website/source/guides/stateful-workloads/stateful-workloads.html.md

```diff
@@ -9,6 +9,14 @@ description: |-
 
 # Stateful Workloads
 
+Nomad allows a user to mount persistent data from local or remote storage volumes
+into task environments in a couple of ways — host volume mounts or Docker Volume
+drivers.
+
```

Comment:
Can you describe the advantage of using host volumes as opposed to using Docker volumes (with the default volume driver)? The example deploys a MySQL server running in Docker. It's not immediately clear what benefit host volumes bring in this case.

Reply:
Thanks for the suggestion; I appreciate it! The primary value proposition is that host volumes are a more taskdriver-agnostic approach to volume handling.  Nomad is also fully aware of them for the purposes of dynamic, implicit constraints around their availability.  I will add some prose at the top to see if I can highlight that. I am hoping to highlight more as we grow the documentation around host volumes as well.

Reply:
We could also add a note on the fact they have an ACL system